DynamicAnalyzer/views/android/environment.py

```diff
@@ -405,7 +432,7 @@ def frida_setup(self):
     def run_frida_server(self):
         """Start Frida Server."""
         check = self.adb_command(['ps'], True)
-        if b'/system/fd_server' in check:
+        if b'fd_server' in check:
```

Comment:
was `/system/` not available for `ps` ?

Reply:
for me the output didn't work with the system in the string. It was just "system" in my output. And as this is just more lax, I think the precision of finding only "fd_server" would be nearly the same, since I don't now any other similar name by default on andoird.